server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+app.get('/', (req, res)=>{
+    res.send("Home Page")
+})
+
+app.get('/ping', (req, res)=>{
+    res.send({"message" : "pong"})
+})
+
+app.listen(port, ()=>{
```

Comment:
It's a good practice to add a catch-all route at the end of all your routes to handle unknown endpoints. This can return a message like "Route not found". [medium]